static/src/js/hooks/useAvailableProviders.js

```diff
@@ -11,10 +11,11 @@ import { GET_AVAILABLE_PROVIDERS } from '../operations/queries/getAvailableProvi
  */
 const useAvailableProviders = () => {
   const { providerId, setProviderId } = useAppContext()
+
   const { user } = useAuthContext()
   const { uid } = user || {}
 
-  const { data: providerData } = useQuery(GET_AVAILABLE_PROVIDERS, {
+  const { data = {} } = useQuery(GET_AVAILABLE_PROVIDERS, {
```

Comment:
If this request errors I don't think this default will work, data will be null. We should probably use the `errors` key and avoid trying to map the providerIds below